### PR TITLE
Soy Milk can now OD.

### PR DESCRIPTION
**Pull request by AcapitalA**
Soy Milk can now OD people at 30u, causing them to turn into your average bugman for the time being.
Spraytan has the same thing for Jersey Jocks, so why not the opposite?

## Changelog
:cl: add: Soy Milk can now turn you into a bugman. /:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -254,6 +255,17 @@
 		. = TRUE
 	..()
 
+/datum/reagent/consumable/soymilk/overdose_process(mob/living/M, delta_time)
+	metabolization_rate = 1 * REAGENTS_METABOLISM
+	if(ishuman(M))
+		if(DT_PROB(5, delta_time))
+			var/static/list/filth = list("silicons", "ethereals", "plasmamen", "moths", "lizards") //Writing the OD speeches was fun, good times.
+			M.say(pick("Opening those soy bottles is hard, i think my strength is waning...", "Honestly, i think [pick (filth)] deserve rights.", "Did you see my profile? I got like 20 thousand likes!", "Did you see the NT announcement where they spoke out about violence against [pick(filth)]? I really should support them more!", "I wish we could get rid of security so there wouldn't be any more violence.", "This place needs more exposed brick walls, maybe even a barcade!"))
```
Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: change this to use pick_list_replacements
see: https://github.com/MisterBones2910/Merchan-TG-uild/tree/master/strings
`M.say(pick_list_replacements(SOY_FILE, "soy"), forced = /datum/reagent/consumable/soymilk)`
search for pick_list_replacements to see examples

Comment (AcapitalA) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: First time working with that, i'll try.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -254,6 +255,17 @@
...
+		if(DT_PROB(3.5, delta_time))
+			M.visible_message(pick("<b>[M]</b> looks a bit pale...", "<b>[M]</b>'s eyes lose focus.", "<b>[M]</b> slouches over.", "<b>[M]</b>'s mouth hangs a bit open."))
```
Comment (TheChosenEvilOne) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
			M.visible_message("<b>[M]</b>[pick(" looks a bit pale...", "'s eyes lose focus.", " slouches over.", "'s mouth hangs a bit open.")]")
```

Comment (TheChosenEvilOne) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: I would prefer if this was an emote call but I am not too sure on adding all of these as emotes.

Comment (AcapitalA) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Oh that's neat, added it, commit soon.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 1 above)
Comment (AcapitalA) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Alright, got it to work, neat.

File: strings/soy_speech.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+    "soy_speech": [
+	"Opening those soy bottles is hard, i think my strength is waning...",
+	"Honestly, i think @pick(filth) deserve rights.",
+	"Did you see my profile? I got like 20 thousand likes!",
+	"Did you see the NT announcement where they spoke out about violence against @pick(filth)? I really should support them more!",
+	"I wish we could get rid of security so there wouldn't be any more violence.",
+	"This place needs more exposed brick walls, maybe even a barcade!"
```
Comment (TheChosenEvilOne) [inline strings/soy_speech.json]: Indentation looks off

Comment (AcapitalA) [reply strings/soy_speech.json]: Hey you're right, for some reason 2 lines were like 4 spaces instead of a tab.

**Comment by TheChosenEvilOne** [comment]
conflict.

**Pull request merged**